packages/@lwc/wds-core/src/browser/wireMockUtil.js

```diff
@@ -0,0 +1,189 @@
+class TestWireAdapterTemplate {
+  static initialValue;
+
+  static emit(value, filterFn) {
+    TestWireAdapterTemplate.initialValue = value;
```

Comment:
I've just realized this pattern may be problematic if two things are mocked for a single test. In that case, the `initialValue` here will conflict between the two mocks. The capture of the initial value should probably be moved from the static `emit` to `buildTestWireAdapter` and attach the `initialValue` there. 

But let's deal with that in a follow-up PR.